assets/app/view/game/player.rb

```diff
@@ -32,6 +33,7 @@ def render
         ]
 
         divs << h(Companies, owner: @player, game: @game, show_hidden: @show_hidden) if @player.companies.any? || @show_hidden
+        divs << h(UnsoldCompanies, owner: @player, game: @game) if @player.unsold_companies.any?
```

Comment:
unless empty?

why is unsold companies needed?

Reply:
During the initial auction you have a pile of companies that you auction off. I wanted to split the display of companies that you are auctioning versus the ones you have bought.